docs/HowTo/clusterconnectivity/klaw-db-connection.md

```diff
@@ -30,7 +30,7 @@ MySQL, Oracle, or PostgreSQL.
 To configure a connection to a relational database management system
 (RDBMS) such as Postgres, Oracle, or MySQL, you need to edit the
 `application.properties` file location in the
-core(klaw/core/src/main/resources) module, with the connection details
+core(Klaw/core/src/main/resources) module, with the connection details
```

Comment:
Changing spelling of "Klaw" should not be part of this PR, since it's for the rule "Klaw Core", can you reverse this one? 

```suggestion
Klaw Core (`klaw/core/src/main/resources`) module, with the connection details
```